src/EditorFeatures/Core/Host/IStreamingFindReferencesPresenter.cs

```diff
@@ -21,7 +21,7 @@ internal interface IStreamingFindUsagesPresenter
         /// search completes <see cref="FindUsagesContext.OnCompletedAsync"/> should be called. 
         /// etc. etc.
         /// </summary>
-        FindUsagesContext StartSearch(string title);
+        FindUsagesContext StartSearch(string title, bool alwaysShowDeclarations);
```

Comment:
Can you doc what this does? It's not obvious without doing a lot of digging (or knowing this exact PR exists.)